src/compile/axis.ts

```diff
@@ -61,6 +61,19 @@ export function parseInnerAxis(channel: Channel, model: Model): VgAxis {
     }
   });
 
+  const props = model.axis(channel).properties || {};
+
+  // it might have more to be included here.
+  ['grid'].forEach(function(group) {
+    const value = properties[group] ?
```

Comment:
Let's leave it as is.   Currently innerAxis is only for grid, but we can support others in the future.  
(That's beyond scope of this PR.)